File: Data Structures & Algorithms/find-duplicate-integer/submission-0.py
class Solution:
    def findDuplicate(self, nums: List[int]) -> int:
        hashmap = {}
        x = 0
        for i in nums:
            if i not in hashmap:
                hashmap[i] = 0
            hashmap[i] +=1

        for i,j in hashmap.items():
            if j>1:
                return i
    
        
        

File: Data Structures & Algorithms/sort-colors/submission-0.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {

        if(nums.size()<1) return;

        int red = 0;
        int blue = 0;
        int white = 0;

        for(int i=0;i<nums.size();i++){
            if(nums[i]==1) blue++;
            if(nums[i]==0) red++;
            if(nums[i]==2) white++;
        }
        for(int i=0;i<red;i++){
            nums[i] = 0;
        } 
        for(int i=red;i<(red+blue);i++) nums[i] =1;
        for(int i=red+blue;i<(red+blue+white);i++) nums[i]=2;
    
    }

};